spec/components/find/courses/training_locations/view_preview.rb

```diff
@@ -73,6 +73,33 @@ def two_study_sites_and_two_placements_salaried
                               provider: Provider.new(provider_code: 'DFE', recruitment_cycle: RecruitmentCycle.current)).decorate
           render Find::Courses::TrainingLocations::View.new(course:)
         end
+
+        def show_training_locations_link_disabled
+          course = Course.new(course_code: 'FIND',
+                              sites: [Site.new, Site.new],
+                              funding: 'salary',
+                              study_sites: [Site.new, Site.new],
+                              provider: Provider.new(provider_code: 'DFE', selectable_school: false, recruitment_cycle: FactoryBot.build(:recruitment_cycle, year: 2025))).decorate
+          render Find::Courses::TrainingLocations::View.new(course:)
+        end
+
+        def show_training_locations_link_enabled
+          course = Course.new(course_code: 'FIND',
+                              sites: [Site.new, Site.new],
+                              funding: 'salary',
+                              study_sites: [Site.new, Site.new],
+                              provider: Provider.new(provider_code: 'DFE', selectable_school: true, recruitment_cycle: FactoryBot.build(:recruitment_cycle, year: 2025))).decorate
+          render Find::Courses::TrainingLocations::View.new(course:)
+        end
+
+        def show_training_locations_link_current
```

Comment:
I don't think this is required. We already show the link by default, and you've covered that version above.

Reply:
That's for the current recruitment cycle, the others are for next recruitment cycle